cobigen-templates/templates-devon4j/src/main/templates/crud_ngrx_client_app/templates/src/app/home/${variables.etoName#lower_case}/components/${variables.etoName#lower_case}-alert/${variables.etoName#lower_case}-alert.component.ts.ftl

```diff
@@ -7,10 +7,10 @@ import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
   styleUrls: ['./${variables.etoName?lower_case}-alert.component.scss'],
 })
 export class ${variables.etoName?cap_first}AlertComponent implements OnInit {
-  message: string = '';
-  title: string = '';
-  cancelButton: string = 'Cancel';
-  acceptButton: string = 'Delete';
+  message = '';
+  title = '';
+  cancelButton = 'Cancel';
+  acceptButton = 'Delete';
```

Comment:
Is there any reason why you removed the types (`string`)? Normally it is a good practice to define the correct types.

Reply:
Santos told me to migrate to the linter from MyThaiStar. The linter said that the type is obviously String and therefore the type should be removed.

Reply:
Okay! Understood, thanks for the explanation